manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -491,3 +491,7 @@ defaults:
   istio_cni:
     enabled: false
     chained: true
+  # applies to mutating and validation webhook
+  webhookConfig:
```

Comment:
any concerns with keeping value at the root? if yes, please suggest where to add this property. Reason I added here is it applies to both all webhooks (for now)